packages/e2e-test-utils/src/create-new-post.js

```diff
@@ -28,12 +28,13 @@ export async function createNewPost( {
 	} ).slice( 1 );
 	await visitAdminPage( 'post-new.php', query );
 
-	await page.evaluate( ( _enableTips ) => {
-		const action = _enableTips ? 'enableTips' : 'disableTips';
-		wp.data.dispatch( 'core/nux' )[ action ]();
-	}, enableTips );
+	const areTipsEnabled = page.evaluate( () => wp.data.select( 'core/nux' ).areTipsEnabled() );
 
-	if ( enableTips ) {
+	if ( enableTips !== areTipsEnabled ) {
+		await page.evaluate( ( _enableTips ) => {
+			const action = _enableTips ? 'enableTips' : 'disableTips';
+			wp.data.dispatch( 'core/nux' )[ action ]();
+		}, enableTips );
 		await page.reload();
```

Comment:
Rebasing https://github.com/WordPress/gutenberg/pull/18775 into this PR caused tests that rely on the post title receiving initial focus to fail. Reloading the page after we first disable the welcome guide seems to fix.

cc. @youknowriad